provider/consumer.py

```diff
@@ -353,8 +353,12 @@ def __fireTrigger(self, messages):
 
                     # Manually commit offset if the trigger was fired successfully. Retry firing the trigger
                     # for a select set of status codes
-                    if status_code == 200:
-                        logging.info("[{}] Fired trigger with activation {}".format(self.trigger, response.json()['activationId']))
+                    if status_code in range(200, 300):
```

Comment:
that might be 299 not 300 if range is inclusive

Reply:
the lower bound is inclusive while the upper bound is exclusive. i checked 😉

Reply:
haha !